src/lib/store-cash/store-cash-cancel.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+import type { Config } from '../../types';
+
+import { storeCashFpti } from './store-cash-fpti';
+
+export const storeCashCancel = (config : Config) => {
```

Comment:
I would reconsider the name of this func and file, it's more of a merchant user file, with a recognized user event.